django/test/selenium.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals
+
+import sys
+import unittest
+
+from django.test import LiveServerTestCase, tag
+from django.utils.module_loading import import_string
+from django.utils.six import with_metaclass
+from django.utils.text import capfirst
+
+
+class SeleniumTestCaseBase(type(LiveServerTestCase)):
+    browsers = []
```

Comment:
I think it's something like:
'browser' contains the first browser to run the tests against and 'browsers' will contain the rest of the browsers if more than one are requested.